src/modes/direct_select.js

```diff
@@ -67,6 +73,7 @@ module.exports = function(ctx, opts) {
         }
       });
       this.on('mouseup', () => true, function() {
+        if (dragging) emitModify();
```

Comment:
What if I haven't moved the mouse at all? Just clicked down, waited a second, released?